spec/hardware/location.fmf

```diff
@@ -12,8 +12,13 @@ description: |
     .. versionchanged:: 1.33
        Added ``location.lab-controller`` into specification.
 
+    .. warning::
+
+        `like` operation is not supported by lab-controller, only `!=` and `==`
```

Comment:
"`like` operation" is an implementation detail of Beaker, let's not leak it. How about something more verbose?

```
.. warning::

   Because of Beaker limitation, ``~`` and ``!~`` operators commonly used with text-like
   HW requirements cannot be used with ``lab-controller``. Only ``==`` and ``!=``
   operators are supported.
```

Reply:
Actually, I'd drop the waring completely, and let's mention the limited set of operators in `implemented-by` link. We need to add one anyway, to mark `lab-controller` is now supported by `beaker` plugin - it would be similar to `virtual` note in https://tmt.readthedocs.io/en/stable/spec/hardware.html#memory. There may be other infrastructures that would support more operators, let's not limit them, just Beaker. And in Beaker, we can check to operator and raise an error for unsupported ones, e.g. https://github.com/teemtee/tmt/pull/3016/files#diff-06f5216200e1c8ddc7476f7e6a34f74600098c33344a4e7fc3a21ff5cfce5985R613